app/components/Chessguessr.tsx

```diff
@@ -96,6 +98,8 @@ export const Chessguessr = ({
   };
 
   const setGuess = () => {};
+  useHotkeys('Backspace', takeback, [currentGuess, fenHistory]);
+  useHotkeys('Enter', submitGuess, [currentGuess]);
```

Comment:
Implementation is easy and could lead to more shortcuts later on if needed.
Arguments are `key`, `function`, `dependencies`.

Dependencies are important since the function will be memoized. Without dependencies it will act like guesses are always empty.

Reply:
Looks good